src/components/Modal.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { StyledModal } from "./Modal.styled";
+import PropTypes from "prop-types";
+
+const Modal = ({
+  isVisible,
+  toggleVisibility,
+  modalHeader,
+  cancelBtnLabel,
+  submitBtnLabel,
+  onSubmit,
+  children,
+}) => {
+  const onCancel = () => {
```

Comment:
if you rename it it will be redundant, you can just pass callback directly to `onClick={onCancel}}`